### PR TITLE
Dock ID change

### PR DESCRIPTION
**Pull request by seancavanaugh**
Dock ID is now a string to represent the MAC Address of a given dock. All requests that previously sent a dockID should now send a MAC Address instead. I looked over the UI, but I couldn't see any changes that needed to be made for this update.

### CODE REVIEW

**Comment by brnsmns13** [comment]
@seancavanaugh This is not the initial commit...

File: app/controllers/api.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ API.checkOut = function (req, res) {
 	// TODO
     //Get string data from card and send to other API - get studentID back
     var cardString = req.body.cardString;
-    var dockID = req.body.dockID;
+    var dockID = req.body.macAddress;
```
Comment (brnsmns13) [inline app/controllers/api.js]: The mac will still be transferred as "dockID" in the json, but will come across as a string instead of a number as before.

**Comment by seancavanaugh** [comment]
@brnsmns13 I always call the first commit on a branch the initial commit, bad habit. Everything should be fixed now.

**Comment by daotoo** [comment]
+1, looks like all the changes were done in the right place now.

File: app/controllers/api.js [hunk 2]
```diff
@@ -114,11 +103,9 @@ API.findAllDocks = function(req, res) {
 };
 
 API.setDockLocation = function (req, res) {
-	// TODO
-	var dockID = req.body.dockID;
+	var dockID = req.body.dockID
```
Comment (brnsmns13) [inline app/controllers/api.js]: Sorry to nitpick, but you need a ";" here.

**Comment by brnsmns13** [comment]
@seancavanaugh No worries, +1 from me once that ```;``` is back in.

**Comment by daotoo** [comment]
+1 again. lol @ commit.

**Pull request merged**